EvenNumber_While.py

```diff
@@ -0,0 +1,16 @@
+##Even numbers
```

Comment:
There are a few problems with this program.
1. The result is wrong for the input i have given
2. Output message says "printing odd number" for the even number program

<img width="569" alt="image" src="https://user-images.githubusercontent.com/38807024/203930856-97a1dcae-4b1a-4f98-b057-ad7269d09050.png">